app/src/androidTest/java/com/github/siela1915/bootcamp/ProfileFragmentTest.java

```diff
@@ -72,6 +78,8 @@ public void loggedInCanEditProfileTest() {
         onView(withId(R.id.profileDisplayNameEdit)).perform(ViewActions.replaceText("Modified NewTest"));
 
         onView(withId(R.id.profileDisplayNameEdit)).check(matches(ViewMatchers.withText("Modified NewTest")));
```

Comment:
It may be more maintainable if the string "Modified NewTest" is stored in a local variable or a static variable instead of hard coding it. :)